sdk/wallet-adapter/wallet-kit-core/src/index.ts

```diff
@@ -52,19 +53,34 @@ export interface WalletKitCore {
 export type SubscribeHandler = (state: WalletKitCoreState) => void;
 export type Unsubscribe = () => void;
 
+const SUI_WALLET_NAME = "Sui Wallet";
+
+function sortWallets(wallets: WalletAdapter[], preferredWallets: string[]) {
```

Comment:
Should we use Array.sort and also sort alphabetically as well?

Reply:
Right now we're sorting by discovery order of the wallets, instead of alphabetically. This is intentional to incentivize wallets to register faster on the window. However I do think we'll want better ways to allow customization of sort in the future..